Packages/io.chainsafe.web3-unity/Runtime/Scripts/ChainConfigEntry.cs

```diff
@@ -42,6 +42,7 @@ public INativeCurrency NativeCurrency
         [field: SerializeField] public string Rpc { get; set; }
         [field: SerializeField] public string Ws { get; set; }
         [field: SerializeField] public string BlockExplorerUrl { get; set; }
+        [field: SerializeField] public string ViemName { get; set; }
```

Comment:
If we're loading this from the .txt file do we need it serialized also can't it just be part of the WebGL package as a part of a config?

Reply:
No because what if our config file is outdated, and the chain is not found in the dictionary, so people might want to still enter it manually.

Reply:
we can have serialized configs referenced in the connection provider, I think that's how we did it for WC